packages/apps/composer-app/dx.yml

```diff
@@ -17,17 +17,14 @@ runtime:
       persistent: true
     edgeFeatures:
       signaling: true
-      # echoReplicator: true
 
   services:
-    edge:
-      # url: wss://edge.dxos.workers.dev/
-      url: wss://edge-main.dxos.workers.dev/
     ice:
       - urls: turn:kube.dxos.org:3478
         username: dxos
         credential: dxos
     iceProviders:
+      # TODO(wittjosiah): Should be edge-production.
       - urls: https://edge-main.dxos.workers.dev/ice
```

Comment:
only in prod, edge-main for main.composer

Reply:
yeah, could be overridden with env vars